ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -502,25 +507,24 @@ void OnScreenKeyboard::createOSKLayout(OSKTypes oskType) {
   SkString uniChar;
   sk_sp<SkTypeface> defaultTypeface;
 
-  unsigned int YscaleFactor=screenSize_.height()/baseScreenSize.height();
   unsigned int rowSize=oskLayout_.keyInfo->size();
   oskLayout_.keyPos->resize(rowSize);
   oskLayout_.siblingInfo->resize(rowSize);
-  SkPoint oskStartpt{ oskLayout_.horizontalStartOffset,
-                     (OSK_KB_VERTICAL_OFFSET*screenSize_.height()*YscaleFactor)};
+  SkPoint oskStartpt{ oskLayout_.horizontalStartOffset,(OSK_KB_VERTICAL_OFFSET*screenSize_.height())};
 
   for (unsigned int rowIndex = 0; rowIndex < rowSize; rowIndex++) {
 
     unsigned int columnSize=oskLayout_.keyInfo->at(rowIndex).size();
     oskLayout_.keyPos->at(rowIndex).resize(columnSize);
     oskLayout_.siblingInfo->at(rowIndex).resize(columnSize);
-    if(!rowIndex ) oskLayout_.defaultFocussIndex.set((columnSize/2),rowIndex);
 
     for (unsigned int columnIndex = 0; columnIndex < columnSize; columnIndex++) {
+       KeyInfo_t &keyInfo=oskLayout_.keyInfo->at(rowIndex).at(columnIndex);
+       keyPosition_t &keyPos=oskLayout_.keyPos->at(rowIndex).at(columnIndex);
```

Comment:
as an improvement point, can we try to make use of operator overloading functionality for these operations
Can be addressed later as well.